proseco/catalog.py

```diff
@@ -199,6 +209,12 @@ def _get_aca_catalog(**kwargs):
     aca.guides = get_guide_catalog(stars=aca.acqs.stars, fids=aca.fids, mons=aca.mons,
                                    img_size=img_size_guide, **kwargs)
 
+    # Set output catalog aca.n_guide to reflect what the number of requested
+    # guide stars as determined in guide star selection processing. The input
```

Comment:
I think maybe the "what" needs to be cut from this sentence for it to read as a sentence, but I also don't think that putting "requested" here helps (that seems to get easily confused with the n_guide call arg again as a kind of "request").

Reply:
Done?